api/v1/storagecluster_types.go

```diff
@@ -105,6 +105,11 @@ type StorageClusterSpec struct {
 	DefaultStorageProfile string `json:"defaultStorageProfile,omitempty"`
 
 	StorageProfiles []StorageProfile `json:"storageProfiles,omitempty"`
+
+	// +kubebuilder:validation:MaxLength=230
```

Comment:
It's advisable to keep the value low so that you can create a storage class with a longer suffix in the future.

Reply:
Hmm ok, what should be the value in that case in your opinion? Something around 200?